SecretSanta/test/SecretSanta.Business.Tests/Controllers/UserManagerTester.cs

```diff
@@ -0,0 +1,75 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SecretSanta.Data;
+using SecretSanta.Business;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace SecretSanta.Business.Tests{
+    [TestClass]
+    public class UserManagerTester{
+
+        [TestMethod]
+        public void Create_WithUser_ReturnsUser()
+        {
+            UserManager manager = new();
+            User? newUser = new User(){Id=2, FirstName = "Yocky", LastName = "Slush"};
+            User? createdUser = manager.Create(newUser);
+            Assert.AreSame(newUser, createdUser);
+        }
+
+        [TestMethod]
+        public void List_WithUserData_ReturnsListOfUsers()
+        {
+            UserManager manager = new UserManager();
+            IEnumerable<User?> users = manager.List();
+            Assert.IsTrue(users.Any());
+        }
+
+        [TestMethod]
+        [DataRow(0)]
+        [DataRow(1)]
+        public void GetItem_WithValidId_ReturnsUser(int id)
+        {
+            UserManager manager = new UserManager();
+            User? foundUser = manager.GetItem(id);
+            Assert.IsNotNull(foundUser);
+        }
+
+        [TestMethod]
+        [DataRow(666)]
+        [DataRow(444)]
+        public void GetItem_WithInvalidId_ReturnsUser(int id)
+        {
+            UserManager manager = new UserManager();
+            User? foundUser = manager.GetItem(id);
+            Assert.IsNull(foundUser);
+        }
+
+        public void Remove_WithValidId_ReturnsTrue(int id)
```

Comment:
These look like good test methods however they need the `[TestMethod]` attribute added above them or they wont get run.
```suggestion
        [TestMethod]
        public void Remove_WithValidId_ReturnsTrue(int id)
```